infra/scripts/lock-modules.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+set -euo pipefail
```

Comment:
Can we add this to the pre-commit step?

Reply:
The pre-commit configuration has been just implemented and described in the description of the present PR 🎉